dsc/pwsh.profile.dsc.resource.json

```diff
@@ -0,0 +1,101 @@
+{
```

Comment:
Not blocking, but eventually this needs to be a .yaml so it can be signed

Reply:
Yes

Reply:
I tried but, DSC had issues loading it. I will take that a separate PR.